src/job.rs

```diff
@@ -8,13 +8,13 @@ use crate::configure::*;
 use crate::constants::*;
 use crate::context::Context;
 use crate::dc_imex::*;
-use crate::dc_mimefactory::*;
 use crate::dc_tools::*;
 use crate::events::Event;
 use crate::imap::*;
 use crate::location;
 use crate::login_param::LoginParam;
 use crate::message::{self, Message, MessageState};
+use crate::mimefactory::*;
```

Comment:
Maybe import `MimeFactory` only for consistency.

Reply:
It's needed for `Loaded::Message` only it seems.

Reply:
done.